pygsp/graphs/nngraphs/sphere.py

```diff
@@ -55,6 +55,7 @@ def __init__(self,
 
             for i in range(self.nb_pts):
                 pts[i] /= np.linalg.norm(pts[i])
+                pts[i] *= (diameter / 2)
```

Comment:
Do this outside of the loop?

Reply:
Yes, this whole loop is a waste. It has been done properly in `sphere-graphs`.